### ISSUE TITLE
Workflow execution completed unsuccessfully

### ISSUE THREAD

**Issue by quliping** [replikation/What_the_Phage#169]
Hellow! I have successfully run wtp using example data on my computer, while the calculation of my own sample seems not finish completely. Here is the wtp report and the screenshot of the results dir. What's wrong? 

**[execution_report.zip](https://github.com/replikation/What_the_Phage/files/9186224/execution_report.zip)**

![微信截图_20220726110604](https://user-images.githubusercontent.com/39583607/180914480-c757e6a0-4056-4c64-9432-feeb976909d4.png)

Here is my command line:
![image](https://user-images.githubusercontent.com/39583607/180914979-cbb7ec5d-0f34-461c-ab63-c55abe1ade96.png)

Awaiting for your reply Thank you very much.

**Comment by mult1fractal** [comment]
yes this bug is super annoying.... I' sorry for the inconveniece... yesterday I had the same bug
vibrant is generating some weird duplicates and the R Markdowntable therfore is failing. I resolved it some time ago, but only covered the possible errors in my examples 


what you can do : 
`nextflow command ......` `--vb -resume dd6f5769-744e-471d-b29a-4da48627e8c8`

the big number behind resume is the workflow session
so you can just resume your run without vibrant 



would it be possible for you to upload the files from
`/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562` so I could search for more cases I have to include in my error handle script?

**Comment by quliping** [comment]
Thanks for your reply. Here is a compressed file of the dir '/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562'

[6f6f680db7108acf3546f8b5623562.tar.gz](https://github.com/replikation/What_the_Phage/files/9188926/6f6f680db7108acf3546f8b5623562.tar.gz)

**Comment by quliping** [comment]
> yes this bug is super annoying.... I' sorry for the inconveniece... yesterday I had the same bug vibrant is generating some weird duplicates and the R Markdowntable therfore is failing. I resolved it some time ago, but only covered the possible errors in my examples
> 
> what you can do : `nextflow command ......` `--vb -resume dd6f5769-744e-471d-b29a-4da48627e8c8`
> 
> the big number behind resume is the workflow session so you can just resume your run without vibrant
> 
> would it be possible for you to upload the files from `/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562` so I could search for more cases I have to include in my error handle script?

Thanks for your reply. Here is a compressed file of the dir '/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562'

[6f6f680db7108acf3546f8b5623562.tar.gz](https://github.com/replikation/What_the_Phage/files/9188945/6f6f680db7108acf3546f8b5623562.tar.gz)

**Comment by quliping** [comment]
> > yes this bug is super annoying.... I' sorry for the inconveniece... yesterday I had the same bug vibrant is generating some weird duplicates and the R Markdowntable therfore is failing. I resolved it some time ago, but only covered the possible errors in my examples
> > what you can do : `nextflow command ......` `--vb -resume dd6f5769-744e-471d-b29a-4da48627e8c8`
> > the big number behind resume is the workflow session so you can just resume your run without vibrant
> > would it be possible for you to upload the files from `/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562` so I could search for more cases I have to include in my error handle script?
> 
> Thanks for your reply. Here is a compressed file of the dir '/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562'
> 
> [6f6f680db7108acf3546f8b5623562.tar.gz](https://github.com/replikation/What_the_Phage/files/9188945/6f6f680db7108acf3546f8b5623562.tar.gz)

**Comment by quliping** [comment]
> yes this bug is super annoying.... I' sorry for the inconveniece... yesterday I had the same bug vibrant is generating some weird duplicates and the R Markdowntable therfore is failing. I resolved it some time ago, but only covered the possible errors in my examples
> 
> what you can do : `nextflow command ......` `--vb -resume dd6f5769-744e-471d-b29a-4da48627e8c8`
> 
> the big number behind resume is the workflow session so you can just resume your run without vibrant
> 
> would it be possible for you to upload the files from `/public/home/qlp/seamount-meta/00_new_workdir/wtpwork/GT1_workdir/7f/6f6f680db7108acf3546f8b5623562` so I could search for more cases I have to include in my error handle script?

Has this bug been solved? And I found the other error, that the 'phigaro_wf:phigaro' also failed. Here is the log:

WARNING: underlay of /etc/localtime required more than 50 (95) bind mounts
Please, create config file using phigaro-setup script

**Comment by mult1fractal** [comment]
Hey,

I'm currently working on the revision of the WtP manuscript as this is more time critical atm.
Next week Monday I will adjust the WtP code accordingly to some of the reviewer's comments. That's also when I will fix the bug you encountered.

As for Phigaro:
I think this is Phigaro-related and not WtP related.
The Phigaro dev helped us with the integration so it is better to ask her directly if she can help

**Comment by mult1fractal** [comment]
ok I found that the error was related to virsorter2:
it generated at some point `nan` values instead of `0` 
and the R report will only work with numeric values (in your execution_report.zip)

I'm currently fixing this and it will be updated by the end of this week
